### PR TITLE
[doc] modify some expression in docs

### PR DESCRIPTION
**Pull request by caojy1998**
## Description

Modify the expression in "Install from source" part for Linux.
Line 63 and 69 in : docs/source/install/index.rst

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: docs/source/install/index.rst [hunk 1]
```diff
@@ -60,13 +60,13 @@ For Fedora/RHEL/CentOS users, run:
 
    sudo yum install -y gcc-c++ python3-devel make cmake
 
-Create conda environment for development, run:
+Create conda environment for development. To see the usage of create script, run:
```
Comment (Rhett-Ying) [inline docs/source/install/index.rst:63]: how about listing the detailed command of creating env for cpu build and gpu build. then remind the user to run `-h` for more usage?

File: docs/source/install/index.rst [hunk 2]
```diff
@@ -60,13 +60,13 @@ For Fedora/RHEL/CentOS users, run:
...
 
 .. code:: bash
 
    bash script/create_dev_conda_env.sh -h
 
-Build the shared library, run:
+Build the shared library. To see the usage of build script, run:
```
[the remark this answers is not in the document]
Comment (caojy1998) [reply docs/source/install/index.rst:69]: OK, I will re upload the doc

**Comment by Rhett-Ying** [comment]
why close this PR? you could just add more commits to this PR.

File: docs/source/install/index.rst [hunk 3]
```diff
@@ -60,13 +60,38 @@ For Fedora/RHEL/CentOS users, run:
 
    sudo yum install -y gcc-c++ python3-devel make cmake
 
-Create conda environment for development, run:
+Create conda environment for development for CPU, run:
```
Comment (Rhett-Ying) [inline docs/source/install/index.rst:63]: `To create a Conda environment for CPU development, please execute the following command:`

File: docs/source/install/index.rst [hunk 4]
```diff
@@ -60,13 +60,38 @@ For Fedora/RHEL/CentOS users, run:
...
+   bash script/create_dev_conda_env.sh -c
+
+To create a Conda environment for GPU development, please execute the following command:
+
+.. code:: bash
+
+   bash script/create_dev_conda_env.sh -g 11.7
+
+
+To see the usage of create script, run:
```
Comment (Rhett-Ying) [inline docs/source/install/index.rst:76]: To further configure the conda environment, please execute the following command for more details:

Comment (caojy1998) [reply docs/source/install/index.rst]: OK, I changed it.

**Review by Rhett-Ying — APPROVED** [review]

**Pull request merged**